### PR TITLE
Use class properties

### PR DESCRIPTION
**Pull request by BrychanOdlum**
Switch to using class properties proposal to lexically bind instead of using explicit bindings within the component's constructor.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["@babel/preset-react"],
+  "plugins": ["@babel/plugin-proposal-class-properties"]
+}
```
Comment (raphaelvigee) [inline .babelrc]: \n

File: src/app/components/Common/Player/Player.jsx [hunk 2]
```diff
@@ -58,32 +46,32 @@ class Player extends React.Component {
...
     }
-  }
+  };
 
-  // eslint-disable-next-line class-methods-use-this
-  handleAddToLibrary() {
-    // console.log('Add to library');
-    // this.props.mk.instance.addToLibrary();
-  }
+  // // eslint-disable-next-line class-methods-use-this
```
Comment (raphaelvigee) [inline src/app/components/Common/Player/Player.jsx]: Lets remove it completely?

**Comment by raphaelvigee** [comment]
Also, instead of `.babelrc`, do we not prefer using the `configureBabel` from `webpack-encore`?

File: .babelrc (hunk 1 above)
Comment (BrychanOdlum) [reply .babelrc]: Good catch, fixed.

File: src/app/components/Common/Player/Player.jsx (hunk 2 above)
Comment (BrychanOdlum) [reply src/app/components/Common/Player/Player.jsx]: Yeah, could do. I'm assuming we will add the method back soon so didn't remove it as a reminder.

**Comment by BrychanOdlum** [comment]
> Also, instead of `.babelrc`, do we not prefer using the `configureBabel` from `webpack-encore`?

@raphaelvigee I think I much prefer a dedicated .babelrc. We really aren't using much from webpack-encore and I'm semi-tempted to remove it in the future in favour of a vanilla babel configuration.

File: src/app/components/Common/Player/Player.jsx (hunk 2 above)
Comment (raphaelvigee) [reply src/app/components/Common/Player/Player.jsx]: Hum, fair enough, but I think that we will remember nonetheless

Comment (BrychanOdlum) [reply src/app/components/Common/Player/Player.jsx]: Yeah, that's fair. We have an issue tracking it. I'll remove it in this PR, give me a second and I'll amend the commit.

**Pull request merged**